### DISCUSSION TITLE
Unable to debug on iOS with Hot Restart

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#17621] [General]
I am using a windows machine trying to debug on an local iPhone. This is on a new project created using the Uno wizard.

**System.Exception:** 'No class inheriting from a valid Application Delegate found in /var/mobile/Containers/Data/Application/22726C2B-A4AF-4272-904E-5DC813154C90/Documents/UnoApp2.content/UnoApp2.dll'

**Comment by WenHong-Lau** [comment]
There isn't any build error. Exception was thrown upon launching the app on the local iPhone

**Reply by MartinZikmund** [reply]
The Application Delegate class should definitely be present, can you package the repro as a zip file and post this as an issue please?

**Reply by jeromelaban** [reply]
@WenHong-Lau Could you also provide a [binlog of your build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by WenHong-Lau** [reply]
Here is the repro. This is a newly made project from the wizard.
[UnoApp2.zip](https://github.com/user-attachments/files/16342070/UnoApp2.zip)

**Reply by MartinZikmund** [reply]
Please also provide a binlog, as Jerome mentioned above - reason is the Hot Restart support is generated via a source generator and we need to see whether it was successfully run

**Reply by WenHong-Lau** [reply]
I have placed all the binlogs in the same zip file

**Reply by MartinZikmund** [reply]
Ah, missed that, thanks!

**Reply by MartinZikmund** [reply]
Created https://github.com/unoplatform/uno/issues/17648 to track this
